lib/elm-test.js

```diff
@@ -350,18 +349,18 @@ function runElmTest() {
 
   return ensurePackagesInstalled(testRootDir)
     .then(function() {
-      // Hard link our existing elm-stuff into the generated code,
-      // to avoid re-downloading and recompiling things we already
-      // just downloaded and compiled.
-      var newElmStuffPath = path.join(newElmPackageDir, "elm-stuff");
-
-      if (!fs.existsSync(newElmStuffPath)) {
-        fs.symlinkSync(
-          path.join(testRootDir, "elm-stuff"),
-          newElmStuffPath,
-          "junction" // Only affects Windows, but necessary for this to work there. See https://github.com/gulpjs/vinyl-fs/issues/210
-        );
-      }
+      // // Hard link our existing elm-stuff into the generated code,
+      // // to avoid re-downloading and recompiling things we already
+      // // just downloaded and compiled.
+      // var newElmStuffPath = path.join(newElmPackageDir, "elm-stuff");
+
+      // if (!fs.existsSync(newElmStuffPath)) {
+      //   fs.symlinkSync(
+      //     path.join(testRootDir, "elm-stuff"),
+      //     newElmStuffPath,
+      //     "junction" // Only affects Windows, but necessary for this to work there. See https://github.com/gulpjs/vinyl-fs/issues/210
+      //   );
+      // }
```

Comment:
The new elm-compiler doesn't seem to like symlinks or something; it seemed to be getting confused about what it needed to rebuild, and the tests would fail to compile.  However, I think just removing this (and not having generated-code share elm-stuff with the tests folder) should be fine because Elm 0.19 compiles so much faster.